Nos.xcodeproj/project.pbxproj

```diff
@@ -925,6 +938,23 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		5B46611829CCB66D008B8E8C /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
```

Comment:
Can you set the input and output paths for this script? I think that will make it only run when Localized changes.

Reply:
I created #211 to do this because I couldn't get it working quickly.